stone/backends/tsd_client.py

```diff
@@ -103,9 +126,31 @@ def generate(self, api):
             t_end = len(template)
             t_ends_with_newline = template[t_end - 1] == '\n'
 
-            self.emit_raw(template[0:r_start] + ('\n' if not r_ends_with_newline else ''))
+            if self.args.import_namespaces:
+                import_template_string = self.args.import_template_string
+                import_from_file = self.args.types_file
+                # /*IMPORT*/
+                i_match = re.search("/\\*%s\\*/" % (import_template_string), template)
```

Comment:
LOL this seems super manual, but I see it used in other places. Have we considered using an actual template language library?